README.md

```diff
@@ -51,7 +51,7 @@ Read our stated [Problems and Goals](problems-and-goals.md) for an in-depth unde
 
 ## Meetings
 
-Panel meetings are held most Tuesdays at 10:00AM Eastern / 4:00PM Central European. Check-in on [gitter](https://gitter.im/solid/data-interoperability-panel) for weekly agenda.
+Panel meetings are held every other Monday at 3:00PM UTC (10:00AM Eastern / 4:00PM Central European). This bi-weekly schedule started on October 24th 2022; the latest meeting date can be found in the [minutes](https://github.com/solid/data-interoperability-panel/blob/master/meetings/). Every meeting will be announced on [gitter](https://gitter.im/solid/data-interoperability-panel), together with its agenda.
```

Comment:
It doesn't make sense to me to say the "bi-weekly schedule started on October 24, 2022" when the (currently) latest meeting minutes are for 2022-11-21, and the next previous are for 2022-03-08.

* If meetings were held on the missing dates, there should be tiny files added that provide or link to whatever notes and/or recordings might still be available.

* If meetings were not held on the missing dates, I suggest saying that the bi-weekly schedule started on 2022-11-21.